gulpfile.js

```diff
@@ -82,7 +82,7 @@ gulp.task('test.unit', ['test.compile'], function(done) {
 gulp.task('test.e2e', ['test.compile'], function(done) {
   var testfile = 'helloworld';
 
-  var dir = __dirname + '/build/e2e';
+  var dir = (__dirname + '/build/e2e').replace(/\\/g, '/');
```

Comment:
Could you add a comment that explains that this is for windows, where backslashes are in __dirname?

Also, please only run the replace on __dirname, not the concatenated expression (it's a bit misleading like this).